content/committees/specification/guide/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Guide to the Jakarta EE Specification Process"
+date: "2021-03-04T00:00:00+00:00"
+layout: "single"
+aliases:
+    - committees/specification/guide/
+---
+
+## Content
+
+- [Introduction]({{< ref "#intro" >}})
+- [Creation Review]({{< ref "#creation" >}})
+- [Plan Review]({{< ref "#plan" >}})
+- [Progress Review]({{< ref "#progress" >}}) *(optional)*
+- [Release Review]({{< ref "#release" >}})
+- [Service Release]({{< ref "#service" >}})
+
+## Introduction {#intro}
+
+This guide is a step-by-step practical guide to help Jakarta EE Specification projects navigate the steps of the Jakarta EE Specification Process ([JESP](https://jakarta.ee/about/jesp/)). 
+It is meant as a helpful tool, so **if in doubt, alway refer to the [JESP](https://jakarta.ee/about/jesp/) for completeness**. Please do not hesitate to contact the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) if you have any questions or comments about anything related to the development of Jakarta EE Specifications.
+
+## Creation Review {#creation}
+
+![Creation Review](JESP_creation-review.png)
+
+* A Jakarta Specification project is created as any other project under the Eclipse Development Process ([EDP](https://www.eclipse.org/projects/dev_process/)).
+* A Jakarta Specification must be approved by the Specification Committee.
+
+### Steps Involved
+
+1. Create a Project Proposal (*).
+2. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[creation review template](https://github.com/jakartaee/specifications/blob/master/creation_review_pr_template.md)**.
+3. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a creation review.
+
+(*) It may be a good idea to socialize the idea with the Specification Committee to solicit support before proceeding with the project creation.
+
+## Plan Review {#plan}
+
+![Plan Review](JESP_plan-review.png)
+
+* Each specification project needs to engage in a Plan Review prior to any extensive development effort in support of a new Specification Version. 
+
+### Steps Involved
+
+1. Submit a Pull Request to the [Jakarta EE Specifications repository](https://github.com/jakartaee/specifications) using the **[plan review template](https://github.com/jakartaee/specifications/blob/master/plan_review_pr_template.md)**.
+2. Send an email to the [Specification Committee](mailto:jakarta.ee-spec@eclipse.org) to request a plan review.
+
+## Progress Review {#progress}
+
+![Progress Review](JESP_progress-review.png)
+
+* A progress review can be initiated by the project when they want to inform the specification committee about their progress, but are not ready for a release yet.
```

Comment:
We should make it clear that a Progress Review is completely optional.  The Project can decide to have one, if they wish to share their progress.  Or, as as you indicated, if nothing has happened in 12 months.  But, we should emphasize somehow that these Progress Reviews are on an "exception basis", they are not part of the normal Spec development process.